.vib/grafana-tempo-query/goss/vars.yaml

```diff
@@ -0,0 +1,6 @@
+binaries:
+  - tempo-query
+root_dir: /opt/bitnami
+directories:
+  - paths:
+    - /opt/bitnami/grafana-tempo-query/bin
```

Comment:
As we are not checking the `grafana-tempo-query` binary, I'd replace this check with a `/opt/bitnami/grafana-tempo-query/bin/tempo-query --help`. That way you would both check the dir and the binary.